### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by Becky449**
- Add Redux Toolkit(npm install react-redux @reduxjs/toolkit).
- 
- Structure your application files using a "feature folder" approach and use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your Redux files. Your files/folder structure could look like this:
- 
- ...
- /src
-     |---/components
-     |---/redux
-         |--- /books
-                 | books.js
-         |--- /categories
-                 | categories.js
-         | configureStore.js
-     | index.js
- Write your book's actions and reducer (in /src/redux/books/books.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- 
- Define action types for adding and removing a book.
- Set the initial state to be an empty array of books.
- Export Action Creators for your actions.
- Write your reducer and export it as default.
- Define state changes for the actions that you created.
- In case of unknown action - return the current state.
- Write your categories actions and reducer (in /src/redux/categories/categories.js) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- 
- Define an action type for checking the status.
- Set the initial state to be an empty array of categories.
- Export Action Creators for your actions.
- Write your reducer and export it as default.
- For the check status action return a string "Under construction".
- In case of unknown action - return the current state.
- Configure the Redux Store (/src/redux/configureStore.js):
- 
- Importing the necessary methods from Redux Toolkit.
- Combine both reducers into a root reducer by using configureStore function.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,5 @@
 <a name="readme-top"></a>
```
Comment (DammyShittu) [inline README.md:2]: - [ ] In the configureStore file you need to import `configureStore` from `@reduxjs/toolkit` and use it to set up your store. Kindly refactor you code accordingly.

- [ ] You are required to also import the categories reducer in the file and add it as a reducer in the store.

  image

File: README.md [hunk 2]
```diff
@@ -1,36 +1,5 @@
...
-<div align="center">
-
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
 <!-- TABLE OF CONTENTS -->
```
Comment (DammyShittu) [inline README.md:4]: - [ ] You have not set up the required action type, creators and reducer for the categories. Kindly do that. What you have currently is an empty file.

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Becky449,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Becky449 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] Implemented all the changes from the previous reviewer :white_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**